app/code/Magento/Catalog/view/frontend/templates/product/image_with_borders.phtml

```diff
@@ -13,7 +13,7 @@
         <img class="<?= $block->escapeHtmlAttr($block->getClass()) ?>"
             <?= $block->escapeHtmlAttr($block->getCustomAttributes()) ?>
             src="<?= $block->escapeUrl($block->getImageUrl()) ?>"
-            max-width="<?= $block->escapeHtmlAttr($block->getWidth()) ?>"
-            max-height="<?= $block->escapeHtmlAttr($block->getHeight()) ?>"
+            width="<?= $block->escapeHtmlAttr($block->getWidth()) ?>"
```

Comment:
Looks like there html attributes hadn’t any effect, so we could just remove them. From another side - width and height might be used by custom extensions that processes images somehow, so maybe it’s better to replace them to data-width and data-height.
What do you think?

Reply:
@ihor-sviziev No the images should gave the html width and height tag.
This allows the browsers to render the page better.
As it already know what for image size to expect.